demos/serverless/deploy.js

```diff
@@ -13,6 +13,7 @@ let useEventBridge = false;
 let enableTerminationProtection = false;
 let disablePrintingLogs = false;
 let chimeEndpoint = 'https://service.chime.aws.amazon.com';
+let chimeSDKMeetingsEndpoint = 'https://service.chime.aws.amazon.com';
```

Comment:
There is a `chimeEndpoint` with same value already, any reasons we need a new one?